app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -1686,13 +1690,14 @@ public boolean onPrepareOptionsMenu(final Menu menu) {
         visible = getAlbum().getSelectedCount() > 0;
         menu.findItem(R.id.action_copy).setVisible(visible);
         menu.findItem(R.id.action_move).setVisible((visible || editMode) && !fav_photos);
+        menu.findItem(R.id.unfavourite_image).setVisible((!albumsMode) && (editMode || visible));
         menu.findItem(R.id.action_add_favourites).setVisible((visible || editMode) && (!albumsMode && !fav_photos));
         menu.findItem(R.id.excludeAlbumButton).setVisible(editMode && !all_photos && albumsMode && !fav_photos);
         menu.findItem(R.id.zipAlbumButton).setVisible(editMode && !all_photos && albumsMode && !fav_photos && !hidden &&
                 getAlbums().getSelectedCount() == 1);
         menu.findItem(R.id.select_all).setVisible(editMode);
         menu.findItem(R.id.delete_action).setVisible((!albumsMode || editMode) && (!all_photos || editMode) &&
-                (!fav_photos || editMode));
```

Comment:
Why was this removed?

Reply:
The delete icon was present in the favourites mode and its function was to unfavourite an image. But as its was a delete icon it was a bit confusing for the user what action will it perform . So i remove this line so that the toolbar will not have the delete icon and added the option to unfavourite image from overflow menu.

Reply:
Maybe you could change it to an unfavourite button instead of a delete. But it would be used in other areas as well I guess.

Reply:
So should I add the option to unfavourite an image to the toolbar or keep it in the overflow menu only?

Reply:
Okay then can you remove this from the menu then?

Reply:
Nope. We will go with the same way then.